Library/Formula/iojs.rb

```diff
@@ -1,4 +1,8 @@
+require "language/javascript"
+
 class Iojs < Formula
+  include Language::JS
```

Comment:
I think we didn't. Yeh, can leave it as an `include` for now and I'll add that to my TODO list.